manifests/limits/fragment.pp

```diff
@@ -10,36 +10,36 @@
   include pam
   include pam::limits
   if $::osfamily == 'Suse' and $::lsbmajdistrelease == '10' {
-    fail('You can not use pam::limits::fragment together with Suse 10.x releases')
```

Comment:
defines/classes should be explicit about what they support and if you use an unsupported system, they should fail.